File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaMunicipio.java
package cuponsmart.modelo.pojo.respuesta;

import java.util.List;
import cuponsmart.modelo.pojo.entidad.Municipio;

public class RespuestaMunicipio extends Mensaje{
    private List<Municipio> contenido;
    
    public List<Municipio> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Municipio> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaMunicipio.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Municipio

class RespuestaMunicipio(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Municipio>? = null
): Mensaje(error, mensaje)